src/components/ApplicationList.tsx

```
@@ -143,25 +145,36 @@ const ApplicationList: React.FC<ApplicationListProps> = ({
+                  <Box
+                    as="button"
+                    onClick={() =>
+                      navigate(
+                        `/previewapplication/${app?.internal_application_id}`
+                      )
+                    } // Add your function here
                     width="100%"
-                    height={53}
-                    padding="20px 8px 16px 16px"
-                    justifyContent="space-between"
                   >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)